changelog.md

```diff
@@ -3,13 +3,16 @@
 We are currently rebuilding parts of Unchained under the hood with a new code structure that helps developers to easily add new resolvers and access the core API's through typescript types.
 
 ## Breaking Changes
+
 - [api] `Mutation.setBaseLanguage` removed, base language now set through env `LANG`
 - [api] `Mutation.setBaseCountry` removed, base language now set through env `COUNTRY`
 - [api] Remove OTP based access to an order
 
 ## Major
+
 - [api] Add Assortment.childrenCount to get a number of child assortments
 - [api] New Query.activeWorkTypes to query for all active work types without introspection
+- [api] New mutation.signPaymentProviderForCheckout to sign generic order payment directly
```

Comment:
Move to a new top level version vNEXT

Reply:
ok, was intending this to be included with v0.61 :)